### DISCUSSION TITLE
Simple example with a solid not impacting a Rwall but crossing it

### DISCUSSION THREAD

**Discussion by galinaDor** [OpenRadioss/OpenRadioss#3241] [Q&A]
I am starting with openRadioss and I am trying a simple drop test with a cube moving with gravity (Z) and encountering a Rwall (Z=0) after 1m. 
I didn`t make any inter block.
In the animation output I see the cube moving downwards (due to the gravity I applied to its external nodes). When it reaches the RWALL instead of bouncing on it, it passes through it as if it was not there. What am I missing?
The RWALL is
 #-  7. RIGID WALLS:
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
/RWALL/PARAL/9
New RWALL 9
#  node_ID     Slide grnod_ID1 grnod_ID2
                   2         0         0
#           D_search                fric            Diameter                ffac       ifq
                  .2                  .1                   0                   0         0
#                 XM                  YM                  ZM
               -1000                 100                 550
#               X_M1                Y_M1                Z_M1
               -1000                 100                   0
#               X_M2                Y_M2                Z_M2
               -1000                -100                 550
#---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|

**Comment by PaulAltair** [comment]
2 things might be wrong, 1st, the rwall is directional, you need to make sure the normal direction is towards your object and 2nd, If your block is starting 1m away, then you need to set D_Search big enough to capture it (D_Search is not working like a 'gap' it is simply defining what entities will be seen by the RWALL) alternatively, you can explicitly include the nodes of your block in a GRNOD referenced on GRNOD_ID1 field

**Comment by galinaDor** [comment]
Thanks very much. Yes, changing direction of the plane it works fine.

**Comment by galinaDor** [comment]
I have another question about my cube impacting a RWALL.
If the cube hits the plane flat (one face parallel to the plane) it does not bounce as it stops completelly.
Only inclining the plane slightly the cube start bouncing after impacting the plane.

**Reply by PaulAltair** [reply]
that doesn't sound unreasonable depending on the material used, any slight misalignment of the cube/plane would induce some rotational energy and cause the impact to be 'bouncier'. Do you want it to bounce? What are your material properties? Can you share your simple model?

**Comment by galinaDor** [comment]
Hello, thanks for the support.
I have made a few changes since my last message. Since I will eventually simulate a helmet drop test, I am starting to learn openRadioss using a simplified situation of a metal cube (soon a rounded surface) hitting a layer of styrene foam on the ground. The cube should represent the dummy metal head form used in certification tests.
I draw a simple "cube" of 80mm per side and a "box" with a 500x500mm base and 40mm thickness.
I positioned the cube 10mm above the foam box in order to speed up the simulation and applied to it an initial velocity in the Z direction to reproduce a 1,5m drop approximately and gravity (now deactivated on the rad file).
I fixed the bottom nodes of the box.
Copying some of the procedures of the cube sliding on two beams and then into a cup tutorial example (if I understood it correctly), I applied gravity and initial velocity only on the surface nodes of the cube.
I changed the Stfac of the interface card to 380 (I am not sure how to calculate the exact value) and in this way there isn't penetration as instead it would happen with a 0 value.
My question concerns both all the major errors that are present in the .rad settings I used ( for example properties and material type and values, gravity and inivel nodes, contact interface) and what should be done to reproduce as similar to reality as possible the situation I am trying to simulate. For example I notice that the foam has a very low strain.
[cubebox24_0000.txt](https://github.com/user-attachments/files/18255202/cubebox24_0000.txt)

I changed the rad extension to txt to attach it.

**Comment by galinaDor** [comment]
Hello, I would like to know if my questions above are too vague and general to have an answer here and if I should try to solve some problems on my own and ask straight clear questions.

**Reply by MarianBulla** [reply]
Hello @galinaDor,
and a happy new year 2025 !

Regarding your model:
- please set the inintial velocity to ALL nodes of your part!
- I have also cleand a little bit, your model (removed the 2nd boundary condition, which was not constraining any nodes).

The model is working fine:

[cube_TYPE7_0000.txt](https://github.com/user-attachments/files/18317749/cube_TYPE7_0000.txt)

Regarding the foam material, I would recomend to use LAW70 or LAW90, which is much better suited for such applications, like helmets.

Hope it helps you ?

Thanks and best regards,
Marian
